### DISCUSSION TITLE
Copilot for students question

### DISCUSSION THREAD

**Discussion by AleksanderGPL** [community/community#31876] [Copilot Conversations] [answered]
Can I use the copilot trial before I get my github student pack ? **Will it auto convert from trial to free for students after i get accepted?**

**Comment by davecheney** [comment, marked as answer]
No, it will not. If you want to use copilot while waiting for your student pack you may sign up for a trail — you may be asked to provide payment details, but if you cancel before the end of your trial you will not be charged. 

Once you have cancelled you trial subscription you can apply your student coupon, then signing up for copilot should show the option for a free educational subscription.
